Content.Server/Arcade/SpaceVillainGame/SpaceVillainArcadeSystem.cs

```diff
@@ -106,5 +110,19 @@ private void OnSVillainPower(EntityUid uid, SpaceVillainArcadeComponent componen
 
         if (_uiSystem.TryGetUi(uid, SpaceVillainArcadeUiKey.Key, out var bui))
             _uiSystem.CloseAll(bui);
+
+        component.ShouldSayThankYou = false;
+    }
+
+    private void OnBoundUIClosed(EntityUid uid, SpaceVillainArcadeComponent component, ref BoundUIClosedEvent args)
```

Comment:
```suggestion
    private void OnBoundUIClosed(EntityzSpaceVillainArcadeComponent> ent, ref BoundUIClosedEvent args)
```
Using `Entity` is required for new code. Do this for all new functions in your PR, not just this one.